README.md

```diff
@@ -1 +1,48 @@
-# portfolio-ameer-hamza
\ No newline at end of file
+---
+layout: default
+---
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+
```

Comment:
- [ ] You should add a title related to your project and a short description of it. Kindly work on that. :writing_hand: